app/templating/summary/question.py

```diff
@@ -39,14 +39,27 @@ def _build_answers(cls, question_schema, answer_schema, answers):
                         key = answer_id if answer_index == 0 else '_'.join([answer_id, str(answer_index)])
                         answer = cls._build_answer(question_schema, answer_schema, answers, answers_iterator, key)
                         summary_answers.append(Answer(answer_schema, answer))
-
             else:
                 answer = cls._build_answer(question_schema, answer_schema, answers, answers_iterator)
-                summary_answers.append(Answer(answer_schema, answer))
+                child_answer_value = cls._find_other_value_in_answers(answer_schema, answer, answers)
+                summary_answers.append(Answer(answer_schema, answer, child_answer_value))
 
         return summary_answers
 
     @classmethod
+    def _find_other_value_in_answers(cls, answer_schema, answer, answers):
+        if answer is None:
+            return None
+        elif answer_schema['type'] == 'Checkbox' or answer_schema['type'] == 'Radio':
+            options = answer_schema['options']
+            for option in options:
+                if option['label'] in [a[0] for a in answer] or option['label'] == answer and 'child_answer_id' in option:
+                    for child_answer in answers:
+                        if 'child_answer_id' in option and child_answer == option['child_answer_id']:
+                            return answers[child_answer]
```

Comment:
I don't think so, child_answer would refer to a dictionary key in this case. The return value would be the value associated with the child_answer key